### ISSUE TITLE
Missing declarations for `graphql-server`

### ISSUE THREAD

**Issue by realStandal** [redwoodjs/redwood#3348]
It doesn't look like declaration files for the `@redwoodjs/graphql-server` made it into this latest version (`0.36`), at least they're not in the right/same spot.

When I use an import from graphql-server:

![image](https://user-images.githubusercontent.com/25166787/132415713-b93b4374-cfa3-4c52-a361-349ce3e91e00.png)

My API package.json:

```yaml
{
  "name": "api",
  "version": "0.0.0",
  "private": true,
  "dependencies": {
    "@redwoodjs/api": "^0.36.3",
    "@redwoodjs/graphql-server": "^0.36.3",
    "nodemailer": "^6.6.3",
    "squirrelly": "^8.0.8",
    "uuid": "^8.3.2"
  },
  "devDependencies": {
    "@types/nodemailer": "^6.4.4",
    "@types/uuid": "^8.3.1"
  }
}
```

graphql-server's package.json:

```yaml
{
  "name": "@redwoodjs/graphql-server",
  "version": "0.36.3",
  "files": [
    "dist",
    "logger",
    "webhooks"
  ],
  "main": "./dist/index.js",
  "types": "./dist/index.d.ts",
  ...
}
```

The declarations referenced aren't actually in the root of the `/dist/` directory, but I did find (at least some) under `/dist/src`. So I was able to update:

 `"types": "./dist/index.d.ts"`

to

 `"types": "./dist/src/index.d.ts"`

and everything's working as it was prior to `0.36`.

**Comment by thedavidprice** [comment]
Looping in @dthyresson 

This is potentially resolved by #3286 and #3307

**Comment by dthyresson** [comment]
> Looping in @dthyresson
> 
> This is potentially resolved by #3286 and #3307

Correct. With the graphql and secure services rework should no longer be an issue.
